### PR TITLE
domain: load stats when stats lease is 0

### PR DESCRIPTION
**Pull request by lamxTyler**
### What problem does this PR solve? 

Sometimes, we need to automatically load stats even if the stats lease is 0.

### What is changed and how it works?

Load stats even when stats lease is 0, and use `3s` as the default interval.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
     Change the stats lease to 0, and the log shows `["init stats info time"] ["take time"=2.187982ms]`

Code changes

 - Has exported function/method change

Side effects

 - None

Related changes

 - Need to cherry-pick to the release branch
 - Need to be included in the release note

### CODE REVIEW

File: domain/domain.go [hunk 1]
```diff
@@ -888,6 +888,8 @@ func (do *Domain) UpdateTableStatsLoop(ctx sessionctx.Context) error {
 	statsHandle := handle.NewHandle(ctx, do.statsLease)
 	atomic.StorePointer(&do.statsHandle, unsafe.Pointer(statsHandle))
 	do.ddl.RegisterEventCh(statsHandle.DDLEventCh())
+	do.wg.Add(1)
+	go do.loadStatsWorker()
```
Comment (coocood) [inline domain/domain.go]: Can we move this out of UpdateTableStatsLoop?

Comment (lamxTyler) [reply domain/domain.go]: Why?

Comment (coocood) [reply domain/domain.go]: never mind it doesn't matter much

**Comment by coocood** [comment]
LGTM

**Pull request merged**